### ISSUE TITLE
v1.8.4 Count of unseen integrations isn't reliably going down on each view.`

### ISSUE THREAD

**Issue by danieliser** [PopupMaker/Popup-Maker#659]
Possibly our function isn't getting run to save what has been viewed, or the data being saved isn't in a useful place (user meta vs options).

Or could just be typos.

**Comment by fpcorso** [comment]
@danieliser Do you know if this is still occurring? Have we had any recent reports or can we close this out?

**Comment by danieliser** [comment]
I think it does. I think I did narrow it down but didn't update the ticket.

Essentially there are 2 ways to get to the extend page, one from the main admin menu, the other from the tabs we insert above the All Popups and Popup Themes screens.

For some reason, these 2 do not use the exact same query args in the url.

The code that dismisses viewed tabs is only running when it detects specific query args, and that does not appear to be the case when you view it from one method vs the other.

So users who go to the tab above all popups screen get them cleared, those that go to Popup Maker -> Extend do not. Or vice versa, but that was what I believe I worked out.

Still needs a patch, but should be fast.

**Comment by fpcorso** [comment]
@danieliser 

I'm not sure if that is still happening. I just tested this with the views still showing all 25 need to be viewed. I first clicked the tab from all popups and viewed the first page. The count in both places changes. I went back to All Popups and this time clicked on Extend in the menu and viewed the second page. The count in both places changed again correctly.

I recorded a loom of my testing. It seems like the count is working as intended regardless of which way the admin gets to that page:
https://www.loom.com/share/5f2c8ffacffe4f44b97afe99dee81db5

Right now, the count is stored as `pum_extend_viewed_extensions` option so it isn't a per user issue. And that option is updated when the page is viewed:
https://github.com/PopupMaker/Popup-Maker/blob/master/classes/Admin/Extend.php#L651

This function is conditionally called by query args; however, it only checks if post_type is `popup` and page is `pum-extensions`. In both cases of clicking through "Extend" or "Addons and Integrations" both of those args are set correctly:
https://github.com/PopupMaker/Popup-Maker/blob/master/classes/Admin/Extend.php#L42
